pkg/metrics/metrics.go

```diff
@@ -0,0 +1,44 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Open Component Model contributors.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package metrics
+
+import (
+	mh "github.com/open-component-model/pkg/metrics"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+const (
+	metricsComponent = "replication_controller"
+)
+
+func init() {
+	metrics.Registry.MustRegister(SubscriptionsReconciledTotal, SubscriptionsReconcileFailed)
+}
+
+// SubscriptionsReconciledTotal counts the number times a subscription was reconciled.
+var SubscriptionsReconciledTotal = mh.MustRegisterCounterVec(
+	"ocm_system",
```

Comment:
we probably shouldn't hardcode the namespace.

Reply:
I mean... the thing about the namespace is that the labels NEED to match later on, right? If we make it configurable, it will result in metrics that might not match existing metrics.

Reply:
Also, the namespace doesn't have to coincide with the actual kubernetes namespace shit's in.